test/integration/test_training.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import pytest
+
+from sagemaker.tensorflow import TensorFlow
+from test.integration.docker_utils import Container
+
+
+@pytest.fixture
+def py_full_version(py_version):
+    if py_version == '2':
+        return '2.7'
+    else:
+        return '3.6'
+
+
+def test_py_versions(docker_image, processor, py_full_version):
+    with Container(docker_image, processor) as c:
```

Comment:
Is it possible to delete test/integration/docker_utils.py and use local mode to test everything?

docker_utils is and older version of local mode.

Reply:
I can remove the training/local_mode part from docker_utils. It's a bit cleaner for this test to just do 'docker run'. I don't need to pass in some fake data.

Reply:
Good call.